trino/__init__.py

```diff
@@ -17,6 +17,6 @@
 from . import exceptions
 from . import logging
 
-__all__ = ['auth', 'dbapi', 'client', 'constants', 'exceptions', 'logging']
+__all__ = ['auth', 'dbapi', 'client', 'constants', 'exceptions', 'logging', 'sqlalchemy']
```

Comment:
This has no effect. the`__all__` only serves as documentation/convention. The `from . import sqlalchemy` seems to be the missing piece but Josh might know better.

Reply:
`__all__` seems to define the meaning for `from trino import *`, so it's kind of part of public API.

i think it's fair to require that `trino.sqlalchemy` be explicitly imported by a user